api/requirements.txt

```diff
@@ -0,0 +1,3 @@
+redis==2.10.3
```

Comment:
The explicit requirements.txt is needed, see the doc of the docker image, there is an auto build